src/ServiceControl/Licensing/LicenseController.cs

```diff
@@ -29,12 +30,22 @@ public async Task<ActionResult<LicenseInfo>> License(bool refresh, CancellationT
                 Status = activeLicense.IsValid ? "valid" : "invalid",
                 LicenseType = activeLicense.Details.LicenseType ?? string.Empty,
                 InstanceName = settings.InstanceName ?? string.Empty,
-                LicenseStatus = activeLicense.Details.Status
+                LicenseStatus = activeLicense.Details.Status,
+                LicenseExtensionUrl = $"https://particular.net/extend-your-trial?p={clientName}{await BuildConnectedApplicationsListPart()}"
```

Comment:
If an old ServicePulse instance calls this method without passing `clientName` in, do we default the value to `ServicePulse`?

```suggestion
                LicenseExtensionUrl = $"https://particular.net/extend-your-trial?p={clientName ?? "ServicePulse"}{await BuildConnectedApplicationsListPart()}"
```